### ISSUE TITLE
HELP: When running to 'cursor.fetch_all()', the eva_server output 'UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 8: invalid start byte'

### ISSUE THREAD

**Issue by Bean95zx** [georgia-tech-db/eva#510]
![image](https://user-images.githubusercontent.com/44997999/204431103-39c1d9d0-f19e-42af-a3b7-285a547090e3.png)
![image](https://user-images.githubusercontent.com/44997999/204431175-9aec0cc0-0a72-4671-b8fa-841ced6424a6.png)

**Comment by jiashenC** [comment]
Can you list more detailed steps how to reproduce this issue? By just using this code snippet, I cannot reproduce your issue.

**Comment by Bean95zx** [comment]
I used the main branch project. I installed version 0.0.10 of evadb in pip.
 I started the "eva_server &" command from the command line, and then executed the code in the image. 
![image](https://user-images.githubusercontent.com/44997999/204949749-933aadee-b4df-470e-b3b8-c9d9f5d62e3a.png)
The server will report this UnicodeDecodeError.
 I use this code is aim to test the abi provided by server from a python project.
 I run eva_cmd__client.py to start the client  will also report this error.  
But I started eva_client in the command is smoothly. I'm confused.

**Comment by Bean95zx** [comment]
next is the all packages in my environment:
absl-py==1.1.0
aiosignal==1.3.1
alembic==1.8.1
antlr4-python3-runtime==4.10
anyio==3.6.2
appdirs==1.4.4
astor==0.8.1
astunparse==1.6.3
async-timeout==3.0.1
attrs==22.1.0
autocfg==0.0.8
backcall==0.2.0
banal==1.0.6
black==22.6.0
blinker==1.4
brotlipy==0.7.0
cached-property
cachetools==4.2.4
certifi 
cffi 
chardet 
charset-normalizer==2.0.12
click 
colorama 
cryptography
cycler==0.11.0
Cython==0.29.30
dataset==1.5.2
debugpy==1.6.4
decorator==5.1.1
dill==0.3.6
diskcache==5.4.0
distlib==0.3.6
django-commands==0.7
easyocr==1.5.0
entrypoints==0.4
evadb==0.0.10
facenet-pytorch==2.5.2
fastapi==0.87.0
fastjsonschema==2.16.2
filelock==3.8.0
flake8==3.9.1
flatbuffers==1.12
fonttools==4.33.3
frozenlist==1.3.3
fsspec==2022.10.0
funcsigs==1.0.2
future==0.18.2
gast==0.3.3
gluoncv==0.10.5.post0
google-auth==1.35.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
graphviz==0.8.4
greenlet==1.1.3
grpcio==1.43.0
grpcio-tools==1.47.0
h11==0.14.0
h5py 
idna==3.3
imageio==2.22.4
importlib-metadata 
importlib-resources==5.10.0
iniconfig==1.1.1
ipykernel==6.16.2
ipython==7.34.0
ipywidgets==8.0.2
isort==5.10.1
jedi==0.18.1
joblib==1.1.0
jsonschema==4.17.1
jupyter_client==7.4.7
jupyter_core==4.12.0
jupyterlab-widgets==3.0.3
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.2
kiwisolver==1.4.3
Mako==1.2.1
Markdown==3.3.7
MarkupSafe==2.1.1
matplotlib==3.5.1
matplotlib-inline==0.1.3
mccabe==0.6.1
mkl-fft==1.3.1
mkl-random==1.2.2
mkl-service==2.4.0
mock==4.0.3
msgpack==1.0.4
multidict 
munkres==1.1.4
mxnet==1.9.1
mypy-extensions==0.4.3
nbclient==0.6.8
nbformat==5.7.0
nbmake==1.3.4
nest-asyncio==1.5.6
netron==5.9.1
networkx==2.6.3
nltk==3.7
numpy==1.21.6
nvidia-cublas-cu117==11.10.1.25
nvidia-cuda-runtime-cu117==11.7.60
nvidia-cudnn-cu116==8.4.0.27
nvidia-tensorrt==8.4.1.5
oauthlib==3.2.0
objgraph==3.5.0
onnx==1.12.0
opencv-python==4.5.5.64
opencv-python-headless==4.5.4.60
opt-einsum==3.3.0
packaging==21.3
pandas==1.3.5
parso==0.8.3
pathlib==1.0.1
pathspec==0.10.1
petastorm==0.11.5
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.0.1
pkgutil_resolve_name==1.3.10
platformdirs==2.5.2
pluggy==1.0.0
portalocker==2.5.1
prettytable==3.3.0
prompt-toolkit==3.0.30
protobuf==3.19.4
psutil==5.9.1
ptyprocess==0.7.0
py==1.11.0
py4j==0.10.9.5
pyarrow==10.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.7.0
pydantic==1.10.2
pyflakes==2.3.1
Pygments==2.12.0
PyMySQL==0.10.1
pyparsing==3.0.9
pyrsistent==0.19.2
pyspark==3.0.2
pytest==7.1.2
python-bidi==0.4.2
python-dateutil==2.8.2
python-multipart==0.0.5
pytools==2022.1.12
pytz==2022.1
PyWavelets==1.3.0
PyYAML==5.1
pyzmq==24.0.1
rarfile==4.0
ray==1.13.0
regex==2022.6.2
requests==2.27.1
requests-oauthlib==1.3.1
rsa==4.8
scikit-image==0.19.3
scipy==1.4.1
seaborn==0.11.2
Shapely==1.8.5.post1
six==1.16.0
sniffio==1.3.0
SQLAlchemy==1.3.20
SQLAlchemy-Utils==0.36.6
starlette==0.21.0
tensorboard==2.3.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.7.0
tensorflow-estimator==2.3.0
tensorflow-gpu==2.3.1
termcolor==2.0.1
thop==0.1.1.post2207130030
tifffile==2021.11.2
tomli==2.0.1
torch==1.10.2
torchaudio==0.11.0+cu113
torchvision==0.11.3
tornado==6.2
tqdm==4.43.0
traitlets==5.3.0
typed-ast==1.5.4
typing_extensions==4.1.1
urllib3==1.26.9
uvicorn==0.20.0
virtualenv==20.17.0
wcwidth==0.2.5
Werkzeug==2.1.2
widgetsnbextension==4.0.3
wrapt==1.14.1
yacs==0.1.8
yarl
zipp==3.8.0

**Comment by gaurav274** [comment]
Thanks for sharing the details. Looks like a mismatch between the server and client versions.  Can you please try running the client script outside the `eva` directory so it uses the `pip` installed version?

Another alternative is to launch the server using `python eva/eva_server.py`.  This will use the server code from the master branch.

**Comment by Bean95zx** [comment]
Yes, I also found the cause of this problem. Because my pip has installed eva, and there is a folder of eva on the master branch. Therefore, when importing packages, there is an error. Therefore, I suggest that the folder name on the main branch can not be called eva
